### PR TITLE
Update description and add Java alternate solution to Cow Gymnastics

### PR DESCRIPTION
**Pull request by Navigator365**

### CODE REVIEW

File: solutions/usaco-963.mdx [hunk 1]
```diff
@@ -132,4 +134,54 @@ int main(){
...
+		PrintWriter pw = new PrintWriter(new FileWriter("gymnastics.out"));
+		StringTokenizer st = new StringTokenizer(r.readLine());
+
+		boolean[][] completePairs = new boolean[20][20];
+		int k = Integer.parseInt(st.nextToken());
+		int n = Integer.parseInt(st.nextToken());
+		int line[] = new int[n];
+		int count = 0;
+
+		while (k != 0) {
```
Comment (SansPapyrus683) [inline solutions/usaco-963.mdx:155]: kinda nitpicky but could you just use a for loop like
```java
for (; k > 0; k--) {
}
```

File: solutions/usaco-963.mdx [hunk 2]
```diff
@@ -132,4 +134,54 @@ int main(){
...
+public class CowGymnastics {
+	public static void main(String[] args) throws IOException {
+		BufferedReader r = new BufferedReader(new FileReader("gymnastics.in"));
+		PrintWriter pw = new PrintWriter(new FileWriter("gymnastics.out"));
+		StringTokenizer st = new StringTokenizer(r.readLine());
+
+		boolean[][] completePairs = new boolean[20][20];
+		int k = Integer.parseInt(st.nextToken());
+		int n = Integer.parseInt(st.nextToken());
+		int line[] = new int[n];
```
Comment (SansPapyrus683) [inline solutions/usaco-963.mdx:152]:
```suggestion
		int[] line = new int[n];
```

File: solutions/usaco-963.mdx [hunk 3]
```diff
@@ -132,4 +134,54 @@ int main(){
...
+import java.io.*;
+import java.util.*;
+
+public class CowGymnastics {
+	public static void main(String[] args) throws IOException {
+		BufferedReader r = new BufferedReader(new FileReader("gymnastics.in"));
+		PrintWriter pw = new PrintWriter(new FileWriter("gymnastics.out"));
+		StringTokenizer st = new StringTokenizer(r.readLine());
+
+		boolean[][] completePairs = new boolean[20][20];
```
Comment (SansPapyrus683) [inline solutions/usaco-963.mdx:149]: put 20 in some constant, not sure what it means here

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
actually really good

File: solutions/usaco-963.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/usaco-963.mdx:155]: or just
```java
for (int i = 0; i < k; i++) {
}
```

Comment (SansPapyrus683) [reply solutions/usaco-963.mdx]: uh really sorry
but could you change it to the second one
it's more intuitive

File: solutions/usaco-963.mdx [hunk 4]
```diff
@@ -81,14 +81,16 @@ public class gymnastics {
 
 We generate the ranking as we read the data.
 
-We create a 2D array of booleans which states if $A$ has a ranking higher than
-$B$ in any point of time. We read every ranking, then for all $\frac{N(N-1)}{2}$
+We create a 2D array of booleans which states if cow $A$ has a ranking higher than
```
Comment (ryanchou-dev) [inline solutions/usaco-963.mdx:84]:
```suggestion
We can create a 2D array of booleans which states if cow $A$ has a ranking higher than
```
does this make it flow better?

File: solutions/usaco-963.mdx [hunk 5]
```diff
@@ -2,7 +2,7 @@
 id: usaco-963
 source: USACO Bronze 2019 December
 title: Cow Gymnastics
-author: Krish Thawani (Java), Hua Zhi Vee (C++)
+author: Krish Thawani (Java), Hua Zhi Vee (C++), Ben Dodge 
 ---
 
 ## Video Solution (Jonathan Paulson)
```
Comment (ryanchou-dev) [inline solutions/usaco-963.mdx:8]:
```suggestion
## Video Solution
```

File: solutions/usaco-963.mdx [hunk 6]
```diff
@@ -81,14 +81,16 @@ public class gymnastics {
 
 We generate the ranking as we read the data.
 
-We create a 2D array of booleans which states if $A$ has a ranking higher than
-$B$ in any point of time. We read every ranking, then for all $\frac{N(N-1)}{2}$
+We create a 2D array of booleans which states if cow $A$ has a ranking higher than
+cow $B$ in any point of time. We read every ranking, then for all $\frac{N(N-1)}{2}$
 pairs we set $b[A][B]=\text{true}$.
```
Comment (ryanchou-dev) [inline solutions/usaco-963.mdx:86]:
```suggestion
cow $B$ in any point of time. After reading every ranking, for all $\frac{N(N-1)}{2}$
pairs we set $b[A][B]=\text{true}$.
```

**Review by ryanchou-dev — COMMENTED** [review]
code is just 😍

File: solutions/usaco-963.mdx (hunk 1 above)
Comment (Navigator365) [reply solutions/usaco-963.mdx]: no problem chief

**Review by SansPapyrus683 — APPROVED** [review]
alr

**Review by nwatx — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**